app/models/user.rb

```diff
@@ -140,6 +143,10 @@ def self.search_states
   scope :active, -> { where(state: 'active') }
   scope :with_valid_password_token, -> { where { lost_password_token_generated_at >= 24.hours.ago } }
 
+  def admin?
```

Comment:
We didn't have that before!? 😳

Reply:
Right! we do!:
https://github.com/3scale/porta/blob/94e6f8ed7aec59e6122566e593e5bbb5220ae152/app/models/user/roles.rb#L14-L18

Good catch 👍